include/io.tcc

```diff
@@ -186,3 +190,128 @@ void IO<Tdim>::write_volumes(const std::map<unsigned, double>& volumes) {
   }
   std::cout << "Wrote Volumes \n";
 }
+
+//! \brief Output .vtk files for viewing material points
+//! \details Write material points coordinates and initial stresses
+//! \tparam Tdim dimension
+template <unsigned Tdim>
+void IO<Tdim>::write_vtk_stresses(
+    const std::vector<Eigen::VectorXd>& coordinates,
```

Comment:
What happens when you don't generate stresses?

Reply:
I think I want it to be zero?

Reply:
Could you check if this would be zero?

Reply:
no, it's not zero... if we don't generate stress, we won't have it...
so I guess we go with checking whether stress is generated first

Reply:
What I will do is that the program will still compute stress and let it be to be zero if `material_properties` are not specified

Reply:
Currently, `write_stresses` would have the same issue as well